models/models.js

```diff
@@ -0,0 +1,2 @@
+export * from './User';
```

Comment:
please rename this file as `index.js` instead of `models.js`, when you need to import `User` and `Product` you just define a folder in path, Node.js will look at index.js in the folder

Reply:
Thanks for the inputs, updated the code to use index.js